spec.md

```diff
@@ -313,6 +313,23 @@ that contains both context and data).
 * Constraints:
   * REQUIRED
 
+### source-labels
+* Type: Map <String, String>
+* Description: Labels associated with the resource that emitted the event.
+  This allows filtering or routing based on non-hierarchical source metadata.
+  This is intended to include (but is not limited to) "labels" in cloud
+  platforms like AWS or Kubernetes.
+* Constraints:
+  * OPTIONAL
+  * Keys MUST match the regular expression `[_.a-z0-9]+`
```

Comment:
Did you mean for the first char to be allowed to be a '.' ?

Reply:
looks like a typo?

Reply:
@inlined did you want to fix this?